File: Java_Program/Classwork/Day_2/Program3.java
///////////////////////////////////////////////////////////////////////////////
//
//  required package
//
//////////////////////////////////////////////////////////////////////////////

import java.util.Scanner;

///////////////////////////////////////////////////////////////////////////////
//
//  Class Name:     CheckEvenOdd
//  Function Name : EvenOdd
//  Description :   it os used to check even or odd
//  Input :         Integer
//  output :        boolean
//  Author :        Rekha Shankarlal Kumawat
//  Date :          10/10/2025
//
///////////////////////////////////////////////////////////////////////////////

class CheckEvenOdd
{
    int iNum  = 0;

    CheckEvenOdd(int iNo1)
    {
        this.iNum = iNo1;
    }

    public boolean evenodd()
    {
        if((iNum % 2) == 0)
    {
        return true;
    }
    else
    {
        return false;
    }
    }

}//End of CheckEvenOdd class

//////////////////////////////////////////////////////////////////////////////
//
//  main class for the application
//
//////////////////////////////////////////////////////////////////////////////

public class Program3 
{
    public static void main(String Arr[])
    {
        Scanner sobj = new Scanner(System.in);

        int iValue = 0;
        boolean bRet = false;

        System.out.println("Enter the Number :");
        iValue = sobj.nextInt();

        CheckEvenOdd cobj = new CheckEvenOdd(iValue);
        bRet = cobj.evenodd();

        if(bRet == true)
        {
            System.out.println(iValue+" is Even Number");
        }
        else
        {
            System.out.println(iValue+" is Odd Number");
        }


    }    
}// End of Main class 

///////////////////////////////////////////////////////////////////////////////
//
//  Input:12                Output:12 is Even Number
//  Input:11                Output:11 is Odd Number
//
///////////////////////////////////////////////////////////////////////////////

File: C++_Program/Classwork/Day_2/program3.cpp
/*   
    Alogrithm
    START
        Accept first number input as iCount
        and Display Jay Ganesh on  screen 
        
    STOP
*/

///////////////////////////////////////////////////////////////////////////////
//
//  required Header files
//
//////////////////////////////////////////////////////////////////////////////
#include<iostream>
 using namespace std; 

///////////////////////////////////////////////////////////////////////////////
//
//  Function Name : Display
//  Description : it to Displat the String Jay Ganesh
//  Input :         Integer
//  output :        void
//  Author :        Rekha Shankarlal Kumawat
//  Date :          10/10/2025
//
///////////////////////////////////////////////////////////////////////////////

void Display(int iFrequency)
{
    int iCnt = 0;

    if(iFrequency < 0)
    {
        cout<<"Invalid input"<<"\n";
    }
    for(iCnt= 1; iCnt <= iFrequency ; iCnt++ )
    {
        cout<<"Jay Ganesh.."<<"\n";
    }
}

///////////////////////////////////////////////////////////////////////////////
//
//Entery point function
//
///////////////////////////////////////////////////////////////////////////////

int main()
{
    int iCount = 0;
    cout<<"Enter the frequency:"<<"\n";
    cin>>iCount;

    Display(iCount);
    return 0;
}

///////////////////////////////////////////////////////////////////////////////
//
//  Input:2                 Output:Jay Ganesh
//                                 Jay Ganesh
//  Input:3                 Output: Jay Ganesh
//                                  Jay Ganesh
//                                  Jay Ganesh
//
///////////////////////////////////////////////////////////////////////////////